internal/gps/cmd.go

```diff
@@ -196,12 +198,31 @@ func runFromRepoDir(ctx context.Context, repo vcs.Repo, timeout time.Duration, c
 	return c.combinedOutput(ctx)
 }
 
+func getenvDuration(key string, def time.Duration) time.Duration {
+	seconds, err := strconv.Atoi(os.Getenv(key))
+
+	if err != nil {
+		return def
+	}
+
+	return time.Duration(seconds) * time.Second
+}
+
 const (
+	// envExpensiveCmdTimeout is the environment variable to read for the
+	// expensiveCmdTimeout default, in seconds.
+	envExpensiveCmdTimeout = "DEP_EXPENSIVE_CMD_TIMEOUT"
```

Comment:
That's a good point, I hadn't thought of that! Personally I'm fine with the environment variable. Let's move forward with that for now. 👍 

The next step is to get some tests that exercise changing the timeout. I don't know how easy/hard that will end up being so if you find yourself going down a rabbit hole, post back here and we can think about it more!